pkg/tsdb/cloudwatch/log_actions.go

```diff
@@ -164,15 +165,15 @@ func (e *cloudWatchExecutor) handleGetLogEvents(ctx context.Context, logsClient
 	}
 
 	messages := make([]*string, 0)
-	timestamps := make([]*int64, 0)
+	timestamps := make([]time.Time, 0)
 
 	sort.Slice(logEvents.Events, func(i, j int) bool {
 		return *(logEvents.Events[i].Timestamp) > *(logEvents.Events[j].Timestamp)
 	})
 
 	for _, event := range logEvents.Events {
 		messages = append(messages, event.Message)
-		timestamps = append(timestamps, event.Timestamp)
+		timestamps = append(timestamps, time.UnixMilli(*event.Timestamp).UTC())
```

Comment:
> .UTC()

Could it be a problem to add UTC() to all the timestamps?

Reply:
Good point! I think in the long run it would be good to also use this function here: https://github.com/grafana/grafana/blob/70f600db1036a1032702a41d7349ea193c765f37/pkg/tsdb/cloudwatch/log_query.go

That function uses something like `time.Parse("2006-01-02 15:04:05.000", *resultField.Value)` to parse times. I guess that can be aligned?

Reply:
I think what you originally wrote is OK since it is milliseconds after UTC in the docs https://docs.aws.amazon.com/AmazonCloudWatchLogs/latest/APIReference/API_OutputLogEvent.html

Reply:
Awesome, thank you for clarifying!